### DISCUSSION TITLE
Running PyTorch locally in Visual Studio Code

### DISCUSSION THREAD

**Discussion by timothydutki** [mrdbourke/pytorch-deep-learning#139] [Q&A] [answered]
By any chance, have you tried running PyTorch in vscode? 
I would very much desire to go through this entire course but I'm very comfortable using pytorch locally through vscode on my Mac. any guidelines on how this can be done would very much be appreciated.

**Comment by mrdbourke** [comment, marked as answer]
Hey @timothydutki,

You can setup PyTorch locally on Mac following the resources here: https://youtu.be/Zx2MHdRgAIc (I made the video)

Also the code version: https://github.com/mrdbourke/pytorch-apple-silicon

Once you've got an environment setup, you can activate it whilst in VSCode and go through the course materials there.

**Reply by timothydutki** [reply]
Thank you, 
I successfully set it up
